app/views/password_capture/new.html.erb

```diff
@@ -1,5 +1,9 @@
 <% title t('titles.visitors.index') %>
 
+<h1 class='h3 my0'>
+  <%= t('headings.passwords.confirm') %>
+</h1>
```

Comment:
do we want this in all cases? Or just in this particular PII case?

Reply:
To be safe, I've made a new route for this password confirmation, so the existing page should be the same. I updated the screenshot in the description as well.

Reply:
After some testing, it may be easier if they use the same route and form, but the form gets some updated content?

Reply:
I think it seems OK to use the same route/form, and show this line optionally, it would probably have to be from data in the session?